js/coinbase.js

```diff
@@ -637,6 +638,360 @@ module.exports = class coinbase extends Exchange {
         return this.parseBalance (result);
     }
 
+    async fetchLedger (code = undefined, since = undefined, limit = undefined, params = {}) {
+        const accountId = this.safeString2 (params, 'account_id', 'accountId');
```

Comment:
We should rework this as shown here:

- https://github.com/ccxt/ccxt/pull/5436/files#diff-3624f6ace8416a4a6ae340743d6421c5R218

Or here

- https://github.com/ccxt/ccxt/blob/master/js/kucoin.js#L1338